Adafruit_FXOS8700.cpp

```diff
@@ -58,23 +58,26 @@
 bool Adafruit_FXOS8700::initialize() {
   Adafruit_BusIO_Register CTRL_REG1(i2c_dev, FXOS8700_REGISTER_CTRL_REG1);
   Adafruit_BusIO_Register CTRL_REG2(i2c_dev, FXOS8700_REGISTER_CTRL_REG2);
-  Adafruit_BusIO_Register XYZ_DATA_CFG(i2c_dev, FXOS8700_REGISTER_XYZ_DATA_CFG);
-  Adafruit_BusIO_Register MCTRL_REG1(i2c_dev, FXOS8700_REGISTER_MCTRL_REG1);
-  Adafruit_BusIO_Register MCTRL_REG2(i2c_dev, FXOS8700_REGISTER_MCTRL_REG2);
+  Adafruit_BusIO_RegisterBits lnoise_bit(&CTRL_REG1, 1, 2);
+  Adafruit_BusIO_RegisterBits mods_bits(&CTRL_REG2, 2, 0);
 
-  /* Set to standby mode (required to make changes to this register) */
-  CTRL_REG1.write(0x00);
+  /* Set the full scale range of the accelerometer */
+  setAccelRange(ACCEL_RANGE_2G);
+
+  /* Low Noise & High accelerometer OSR resolution */
+  standby(true);
+  lnoise_bit.write(0x04);
```

Comment:
I think this should just be `0x01`? You're setting a single bit.
![image](https://user-images.githubusercontent.com/8755041/134250251-5e4ca748-ac40-499c-8700-4c630f32a105.png)

Reply:
Thanks, yep this stems from my original misunderstanding on how RegisterBits worked as a relative (instead of absolute) setter. I thought the `.write()` method would take in a byte and only read the length of bits at that shifted location.

Changed to 0x01!